rkt/run.go

```diff
@@ -376,6 +377,11 @@ func runRun(cmd *cobra.Command, args []string) (exit int) {
 		stderr.PrintE("cannot get the pod manifest", err)
 		return 254
 	}
+
+	if len(manifest.Apps) < 1 {
+		stderr.PrintE("pod has no apps defined", errors.New("you need to define at least one app."))
```

Comment:
We try to avoid addressing users as "you", preferring an indirect wording instead. Can you please reword and get rid of the the `errors.New` too?

Something like:
```
stderr.Print("pod must contain at least one application")
```